evm/src/cpu/cpu_stark.rs

```diff
@@ -44,3 +44,39 @@ impl<F: RichField + Extendable<D>, const D: usize> Stark<F, D> for CpuStark<F, D
         vec![PermutationPair::singletons(8, 9)]
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use anyhow::Result;
+    use plonky2::plonk::config::{GenericConfig, PoseidonGoldilocksConfig};
+
+    use crate::cpu::cpu_stark::CpuStark;
+    use crate::stark_testing::{test_stark_circuit_constraints, test_stark_low_degree};
+
+    #[test]
+    #[should_panic] // TODO: fix this when constraints are no longer all 0.
```

Comment:
nit: maybe best to `#[ignore]` so it shows as ignored rather than passing

Reply:
Good point, done